server/container/guice/guice-common/src/main/java/org/apache/james/utils/MailRepositoryProbeImpl.java

```diff
@@ -34,6 +34,9 @@ public MailRepositoryProbeImpl(MailRepositoryStore repositoryStore) {
         this.repositoryStore = repositoryStore;
     }
 
+    /**
+     * Get the count of email currently stored in a given repository
```

Comment:
now, the method name doesn't match anymore. what about getRepositoryMailCount ?

Reply:
:+1: